src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java

```diff
@@ -62,6 +62,7 @@ public StartEventResponse startEvent(IdamTokens idamTokens, Long caseId, String
     }
 
     public CaseDetails submitEventForCaseworker(IdamTokens idamTokens, Long caseId, CaseDataContent caseDataContent) {
+        log.info("Submitting CCD event for caseworker: {}", caseDataContent.getEvent().getDescription());
```

Comment:
can we add a caseID as well please in the log?

Reply:
None of the other logs in the file do that, they don't even specify what the event is...

Reply:
Tbh this could just be removed as I had it in for testing